modules/nf-core/kmcp/profile/tests/main.nf.test

```diff
@@ -0,0 +1,109 @@
+nextflow_process {
+
+    name "Test Process KMCP_PROFILE"
+    script "../main.nf"
+    process "KMCP_PROFILE"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "kmcp"
+    tag "kmcp/profile"
+    tag "untar"
+    tag "kmcp/compute"
+    tag "kmcp/index"
+    tag "kmcp/search"
+
+    setup {
+        run("UNTAR") {
+            script "../../../untar/main.nf"
+            process {
+                """
+                input[0] = [
+                                [id: 'test'],
+                                file(params.modules_testdata_base_path + 'delete_me/kmcp/kmcp_profile.tar.gz', checkIfExists: true)
```

Comment:
Where is this file coming from? Do we need to put it somewhere better in test_data?

Reply:
There isn't a better place unfortunately. It's only useable for this particular module/toolkit